.storybook/components/Docs/Recipes.stories.mdx

```diff
@@ -2,20 +2,8 @@
 
 # Recipes
 
-Recipes are code examples that show EDS components in context of a larger feature.
+From [Brad Frost](https://bradfrost.com/blog/post/design-system-components-recipes-and-snowflakes/):
 
-## How does a feature become a recipe?
+> Recipe components are specific compositions of design system components (for the most part) that are to be consistently used across a product, but aren’t agnostic enough to live in the design system.
 
-Recipes come from pilots where the EDS team designs a feature or page that will use one or more EDS components. Those pilots are undertaken specifically for the purpose of creating new EDS components and patterns. The EDS team does not implement the resulting design in the product surface area; instead, the EDS team creates the components and also creates recipes showing the components in action. The product team can then use the recipes as a examples for how to implement the feature themselves.
-
-## Does every feature that uses EDS in a product become a recipe?
-
-No, not every feature that uses an EDS component becomes a recipe. If a product team is developing a feature or page on their own and uses an EDS component, the EDS team does not add that feature as a recipe in EDS. Recipes only come from features that the EDS team designs (or codesigns with another team).
-
-## Are recipes imported and used in other products like components?
-
-No, recipes are not used directly in other products. They can be used as examples of how to implement a component on the code side or as general examples of how components can be used in a user interface, but that's all.
-
-## What functions do recipes perform?
-
-Aside from serving as examples for how components can be used (in a user interface or in a code context), they also serve as tests to verify that the components continue to work in the context of a larger feature or page.
+Your team may have some recipes already! To learn more about those, refer to your team's tab in [ZeroHeight](https://eds.czi.design/). The page will contain themed recipes specific to your product. If curious, you can find a similar page for other teams' recipes as well.
```

Comment:
What about some more inclusive language for point solutions folks in the front of the sentence more i.e. "refer to your team or another team's tab to view examples" or something... I think one of the feedback we got from the surveys was trying to find code examples to copy paste for their own use

Reply:
Actually this makes me think of a gap we have. The individual teams will have their recipes in their own storybooks (this is the normal eng act of building product components). We want to avoid work that forces us to manually sync their work with the global zeroheight. 

I'll create a ticket for this, and we can discuss more next week. For now, I'm thinking to remove this line, and then we can add a bit of description on where to look for coded recipes, etc. once we chat about it

Reply:
Makes sense to me

Reply:
@jinlee93 took this part out, and added a note around this to our project ticket